packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -66,8 +67,8 @@ class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasse
                     className={get(this.props, "managedClasses.checkbox_input")}
                     type="checkbox"
                     ref={this.inputRef}
-                    onChange={this.props.onChange}
-                    checked={this.props.checked}
+                    onChange={this.handleCheckboxChange}
```

Comment:
What will happen if I want to know when this has changed? I can no longer pass an onChange here because its being controlled by the checkbox.

Reply:
Adding back in

Reply:
Looks like you need to update jest tests after adding this back in.

Reply:
Should be good now.